deepchecks/utils/dataframes.py

```diff
@@ -27,6 +27,7 @@
 
 def default_fill_na_per_column_type(df: pd.DataFrame, cat_features: t.Union[pd.Series, t.List]) -> pd.DataFrame:
     """Fill NaN values per column type."""
+    pd.set_option('mode.chained_assignment', None)
```

Comment:
maybe put this in the general deepchecks init

Reply:
I prefer to know about these things, so prefer not bo block categorically